Makefile

```diff
@@ -85,9 +85,12 @@ endif
 	mkdir -p build
 	touch $@
 
-oldtest: | nvim
+oldtest: | nvim tags
```

Comment:
Hard-coding a help tags fragment in the test and saving it with `helpers.write_file()` would be more than fine.